core/com.b2international.snowowl.datastore.server/src/com/b2international/snowowl/datastore/server/version/PublishManager.java

```diff
@@ -49,60 +46,74 @@
 import com.b2international.snowowl.core.branch.Branch;
 import com.b2international.snowowl.datastore.BranchPathUtils;
 import com.b2international.snowowl.datastore.CDOEditingContext;
+import com.b2international.snowowl.datastore.CodeSystemVersionEntry;
 import com.b2international.snowowl.datastore.CodeSystemVersions;
-import com.b2international.snowowl.datastore.ICodeSystemVersion;
+import com.b2international.snowowl.datastore.cdo.CDOServerCommitBuilder;
 import com.b2international.snowowl.datastore.cdo.CDOUtils;
-import com.b2international.snowowl.datastore.cdo.ICDOTransactionAggregator;
+import com.b2international.snowowl.datastore.oplock.impl.DatastoreLockContextDescriptions;
 import com.b2international.snowowl.datastore.version.IPublishManager;
 import com.b2international.snowowl.datastore.version.PublishOperationConfiguration;
 import com.b2international.snowowl.eventbus.IEventBus;
-import com.b2international.snowowl.terminologymetadata.CodeSystem;
 import com.b2international.snowowl.terminologymetadata.CodeSystemVersion;
 import com.b2international.snowowl.terminologyregistry.core.builder.CodeSystemVersionBuilder;
 import com.b2international.snowowl.terminologyregistry.core.request.CodeSystemRequests;
-import com.google.common.base.Supplier;
-import com.google.common.base.Suppliers;
 import com.google.common.collect.Iterables;
 
 
 /**
  * Abstract component publish manager implementation.
- * 
- *
  */
 public abstract class PublishManager implements IPublishManager {
 
 	/** Shared logger instance. */
 	protected static final Logger LOGGER = getLogger(PublishManager.class);
-
-	private Supplier<CDOEditingContext> editingContextSupplier;
+	
+	private static final String NEW_VERSION_COMMIT_COMMENT_TEMPLATE = "Created new version '%s' for %s.";
+	private static final String ADJUST_EFFECTIVE_TIME_COMMIT_COMMENT_TEMPLATE = "Adjusted effective time to '%s' for %s version '%s'.";
```

Comment:
This comment doesn't seem to be in use any more, with the section in `getCommitComment` commented out.